src/assets/style/typography.scss

```diff
@@ -5,7 +5,10 @@
 body,
 *,
 html {
-  font-size: 10px;
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+  font-size: 62.5%;
```

Comment:
Rozumiem zamysł i jest to dobry pomysł z zmianą typografi lecz proszę wpierw zapytać czy taka zmiana będzie akceptowana.